platform/platform-app/src/main/resources/features.xml

```diff
@@ -808,6 +808,7 @@
         <feature prerequisite="true" version="${karaf.version}">eventadmin</feature>
         <feature prerequisite="true" version="${karaf.version}">jasypt-encryption</feature>
         <feature prerequisite="true" version="${karaf.version}">http</feature>
+        <bundle>mvn:ddf.lib/common-system/${project.version}</bundle>
```

Comment:
❓ I thought this was going to be moved elsewhere?

Reply:
Yes, that was in one of my later commits. Fixed